<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMovies</class>
 <widget class="QDialog" name="AddMovies">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Movie Information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Genre Information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Studio Information</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_movie_id">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	padding-left: 10px;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_movie_title">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	padding-left: 10px;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_movie_year">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	padding-left: 10px;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Release Year</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_genre_id">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	padding-left: 10px;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Genre ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_genre_name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	padding-left: 10px;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Genre Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_studio_name">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	padding-left: 10px;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Studio Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_studio_id">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	padding-left: 10px;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Studio ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_founded_year">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	padding-left: 10px;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Founded Year</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_headquarters">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	padding-left: 10px;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Headquarters</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddMovies</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddMovies</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
